tests/test_date_filter.py

```diff
@@ -74,19 +74,19 @@ def test_valid_datefilter(self):
         """ Test filtering on dates """
         self.assertEqual(len(self.queryset), 1)
 
-        req1 = Request(self.factory.get('/foo/', {'occurred_max': self.a_date}))
+        req1 = self.factory.get('/foo/', {'occurred_max': self.a_date})
```

Comment:
Changes to these tests all reflect the API changes in DRF 3.8 and django-filter 1.0+ that were implemented in the app code above.